src/fides/api/oauth/utils.py

```diff
@@ -160,6 +133,98 @@ def _get_request_task_jwe_or_error(
     return token
 
 
+def validate_download_token(token: str, privacy_request_id: str) -> DownloadTokenJWE:
+    """
+    Validate a download token for accessing privacy request packages.
+
+    Args:
+        token: The JWE token to validate
+        privacy_request_id: The privacy request ID the token should grant access to
+
+    Returns:
+        The validated DownloadTokenJWE object
+
+    Raises:
+        AuthenticationError: If token is invalid or expired
+        AuthorizationError: If token doesn't grant access to the requested privacy request
+    """
+    if not token:
+        raise AuthenticationError(detail="Download token is required")
+
+    # Check if token looks like a JWE (should have 5 parts separated by dots)
+    if not token or token.count(".") != 4:
```

Comment:
We already check for token in the previous statement
```suggestion
    if token.count(".") != 4:
```